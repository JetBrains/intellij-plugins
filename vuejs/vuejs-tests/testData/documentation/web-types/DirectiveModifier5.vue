<template>
  <div v-cool:foo.foo.ba456<caret>></div>
</template>
