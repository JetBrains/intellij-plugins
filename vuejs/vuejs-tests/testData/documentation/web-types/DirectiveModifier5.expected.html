<div class='definition'><pre>ba###</pre></div>
<div class='content'>
<p>Bar is pattern with static prefix</p>
</div><table class='sections'>
<tr><td valign='top' class='section'><p>Pattern:</td><td valign='top'>\ba[0-9]{3}\</td>
<tr><td valign='top' class='section'><p>Library:</td><td valign='top'><p>test-lib@2.0.0-rc.24</td>
</table>